### PR TITLE
Addressed the Bug 264632 along with print 12 all cmdlets.

### PR DESCRIPTION
**Pull request by Mohd-Aslam-Shaikh**
Addressed the Bug 264632 along with print 12 all cmdlets.

### CODE REVIEW

File: HPE3PARPSToolkit-CLI.psm1 [hunk 1]
```diff
@@ -7882,7 +7884,7 @@ Function New-3parGroupVVCopy
 	}
 	else
 	{
-		return "Please Select Atlist one from P R or Halt"
+		return "Please Select At-least One from P R or Halt"
```
Comment (harikumar-r) [inline HPE3PARPSToolkit-CLI.psm1]: Change the return string as "To proceed, select either PR or Halt"
But looks like this sentence is in complete. Lets discuss it before finalising the sentence

File: HPE3PARPSToolkit-CLI.psm1 [hunk 2]
```diff
@@ -17604,10 +17628,10 @@ Function Find-3parCage
 		}	
 	}	
 	$Result = Invoke-3parCLICmd -Connection $SANConnection -cmds  $cmd	
-	write-debuglog "  executing Find-3parCage Command , surface scans or diagnostics on physical disks with the command --> $cmd " "INFO:" 	
+	write-debuglog "  executing Find-3parCage Command , surface scans or diagnostics on physical disks with the command   " "INFO:" 	
 	if([string]::IsNullOrEmpty($Result))
 	{
-		return  "SUCCESS : Find-3parCage Command Executed Sucessfully $Result"
+		return  "SUCCESS : Find-3parCage Command Executed Successfully $Result"
```
Comment (harikumar-r) [inline HPE3PARPSToolkit-CLI.psm1]: In the most of the return messages, each word marked s capital letter. Not required this. Please re-check all the return strings.
So, here the return sentences as can "Success : Find-3parCage cmdlet executed successfully $Result"

File: HPE3PARPSToolkit-CLI.psm1 [hunk 3]
```diff
@@ -17737,7 +17761,7 @@ Function Set-3parCage
 		return "ERROR: -CageName is Mandate For the Command to Execute"
```
Comment (harikumar-r) [inline HPE3PARPSToolkit-CLI.psm1]: Here too, the return statement should be "Error: -CageName is a required parameter"

File: HPE3PARPSToolkit-CLI.psm1 [hunk 4]
```diff
@@ -25450,7 +25474,7 @@ Function Set-3parRCopyTargetName
 		return "Error :  -TargetName is Mandate. "
```
Comment (harikumar-r) [inline HPE3PARPSToolkit-CLI.psm1]: Lots of places, you have given the return error message as "Error : <ParameterName> is Mandate" . 
Please replace these error return message to"Error: <ParameterName> is mandatory"

File: HPE3PARPSToolkit-CLI.psm1 [hunk 5]
```diff
@@ -34380,11 +34399,8961 @@ Function Stop-3parWsapi()
...
+
+
+##########################################################################
+######################### FUNCTION Set-3parDomain #########################
+##########################################################################
+Function Set-3parDomain()
+{
+<#
+  .SYNOPSIS
+   Set-3parDomain Change currentdomain CLI environment parameter.
```
Comment (harikumar-r) [inline HPE3PARPSToolkit-CLI.psm1]: Showing currentdomain as a single word. Change it to "current domain"
Corrected sentence: Set-3parDomain changes the current domain CLI environment parameter.

File: HPE3PARPSToolkit-CLI.psm1 [hunk 6]
```diff
@@ -34380,11 +34399,8961 @@ Function Stop-3parWsapi()
...
+
+  .DESCRIPTION
+   The Set-3parDomain command changes the currentdomain CLI environment parameter.
```
Comment (harikumar-r) [inline HPE3PARPSToolkit-CLI.psm1]: same here too

File: HPE3PARPSToolkit-CLI.psm1 (hunk 1 above)
Comment (Mohd-Aslam-Shaikh) [reply HPE3PARPSToolkit-CLI.psm1]: Updated.

File: HPE3PARPSToolkit-CLI.psm1 (hunk 2 above)
Comment (Mohd-Aslam-Shaikh) [reply HPE3PARPSToolkit-CLI.psm1]: Updated.

File: HPE3PARPSToolkit-CLI.psm1 (hunk 3 above)
Comment (Mohd-Aslam-Shaikh) [reply HPE3PARPSToolkit-CLI.psm1]: Updated..

File: HPE3PARPSToolkit-CLI.psm1 (hunk 4 above)
Comment (Mohd-Aslam-Shaikh) [reply HPE3PARPSToolkit-CLI.psm1]: Updated All.

File: HPE3PARPSToolkit-CLI.psm1 (hunk 5 above)
Comment (Mohd-Aslam-Shaikh) [reply HPE3PARPSToolkit-CLI.psm1]: Updated.

File: HPE3PARPSToolkit-CLI.psm1 (hunk 6 above)
Comment (Mohd-Aslam-Shaikh) [reply HPE3PARPSToolkit-CLI.psm1]: Updated..

**Comment by Mohd-Aslam-Shaikh** [comment]
Unable to resolve conflict...
creating new branch..

**Comment by prablr79** [comment]
can you please close this review ? its delayed so long.  we need to test the new array change as well.

**Comment by Mohd-Aslam-Shaikh** [comment]
New PR is open

**Pull request closed without merging**